openrest4j-api/src/main/java/com/openrest/v1_1/ContactlessDineIn.java

```diff
@@ -0,0 +1,42 @@
+package com.openrest.v1_1;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import java.io.Serializable;
+import java.util.*;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ContactlessDineIn implements Serializable, Cloneable {
+    private static final long serialVersionUID = 1L;
+
+    /** Default constructor for JSON deserialization. */
+    public ContactlessDineIn() {}
+
+    public ContactlessDineIn(String comment) {
+        this.comment = comment;
```

Comment:
In the Order, we should get 2 fields: label and labelValue

Reply:
u are right, replacing the fields